docs/streams/upgrade-guide.html

```diff
@@ -53,17 +53,19 @@ <h1>Upgrade Guide and API Changes</h1>
     </ul>
 
     <p>
-        Starting in Kafka Streams 2.6.x, a new processing mode is available, named EOS version 2, which is configurable by setting 
-        <code>processing.guarantee</code> to <code>"exactly_once_beta"</code>.
-        <b>NOTE:</b> The <code>"exactly_once_beta"</code> processing mode is ready for production (<i>i.e.</i>, it's not "beta" software). 
+        Starting in Kafka Streams 2.6.x, a new processing mode is available, named EOS version 2. This can be configured
+        by setting <code>StreamsConfig.PROCESSING_GUARANTEE</code> to <code>StreamsConfig.EXACTLY_ONCE_V2</code> for
+        application versions 3.0+, or setting it to <code>StreamsConfig.EXACTLY_ONCE_BETA</code> for versions between 2.6 and 3.0.
         To use this new feature, your brokers must be on version 2.5.x or newer.
-        A switch from <code>"exactly_once"</code> to <code>"exactly_once_beta"</code> (or the other way around) is
-        only possible if the application is on version 2.6.x.
-        If you want to upgrade your application from an older version and enable this feature,
-        you first need to upgrade your application to version 2.6.x, staying on <code>"exactly_once"</code>,
-        and then do second round of rolling bounces to switch to <code>"exactly_once_beta"</code>.
-        For a downgrade, do the reverse: first switch the config from <code>"exactly_once_beta"</code> to
-        <code>"exactly_once"</code> to disable the feature in your 2.6.x application.
+        If you want to upgrade your EOS application from an older version and enable this feature,
+        you first need to upgrade your application to version 2.6.x, staying on <code>StreamsConfig.EXACTLY_ONCE</code>,
```

Comment:
`you first need to upgrade your application to version 2.6.x`

Why would that be required? On can go to `3.0` directly from my understanding?

Reply:
I just copied over the old instructions for upgrading to beta, and then mentioned how to upgrade to V2 down below. But since V2 is now the "real" version I suppose it should be the main one mentioned here